### ISSUE TITLE
Neural Style Transfer Tutorial typo 

### ISSUE THREAD

**Issue by Marceliny** [pytorch/pytorch#34010]
## 📚 Documentation

In style transfer tutorial, under the [gradient descent](https://pytorch.org/tutorials/advanced/neural_style_tutorial.html#gradient-descent) section, second paragraph's last sentence has a typo that 'module' is written as 'modul'.

The last sentence, 
> The optimizer requires a “closure” function, which reevaluates the modul and returns the loss.

needs to be 
> The optimizer requires a “closure” function, which reevaluates the module and returns the loss.

**Comment by skmatz** [comment]
@Marceliny

Please post any tutorials issues to https://github.com/pytorch/tutorials.

**Comment by mathemage** [comment]
Hi @Marceliny and @skmatz !

I've opened the PR to fix the typo: https://github.com/pytorch/tutorials/pull/879

Thanks for a tip!
Regards,
@mathemage
